pyvo/registry/regtap.py

```diff
@@ -314,25 +486,120 @@ def standard_id(self):
         """
         return self.get("standard_id", decode=True)
```

Comment:
Shouldn't the method for `standard_id` be similar to `access_url`, where it returns the first element of `standard_ids` if it has one element, otherwise it fail?

An example code like:
```python
services = pyvo.regsearch(servicetype='tap', keywords=['heasarc'])
print(services[0].describe())
```
fails  in `describe` because `self.standard_id` is `None`. 

If `def standard_id` is modified to something like `access_url`:
```python
    @property
    def standard_id(self):
        """
        the IVOA standard identifier
        """
        #return self.get("standard_id", decode=True)
        standard_ids = list(set(self["standard_ids"]))
        if len(standard_ids)==1:
            return standard_ids[0]
        else:
            raise dalq.DALQueryError(
                "No unique standard_id.  Use get_service.")
```
and the call to `standard_id` in `describe` is changed to:
```python
# stdid = self.get("standard_id", decode=True).lower()
stdid = self.standard_id
```
Things seem to work.